src/DedicatedHsm/docs/New-AzDedicatedHsm.md

```diff
@@ -152,7 +184,7 @@ Accept wildcard characters: False
 SKU of the dedicated HSM
 
 ```yaml
-Type: System.String
+Type: Microsoft.Azure.PowerShell.Cmdlets.DedicatedHsm.Support.SkuName
```

Comment:
Changing the type from string to SkuName, is it a breaking change?

Reply:
+ Sku [old swagger](https://github.com/Azure/azure-rest-api-specs/blob/3dddb7090671b48ce3f569557ed8a1c6ba36c2a7/specification/hardwaresecuritymodules/resource-manager/Microsoft.HardwareSecurityModules/preview/2018-10-31-preview/dedicatedhsm.json#L486) 
+ Sku [new swagger](https://github.com/Azure/azure-rest-api-specs/blob/f203c1b73639b7e2f85aa96902cfe36c6dd2ffad/specification/hardwaresecuritymodules/resource-manager/Microsoft.HardwareSecurityModules/stable/2021-11-30/dedicatedhsm.json#L566).

The sku of the old swagger be defined enum and value has only one. so it's type as string be generated via autorest.
I think it's not a breaking change.